kotlinx-coroutines-core/common/src/flow/operators/Transform.kt

```diff
@@ -34,6 +35,11 @@ public inline fun <T> Flow<T>.filterNot(crossinline predicate: suspend (T) -> Bo
 @Suppress("UNCHECKED_CAST")
 public inline fun <reified R> Flow<*>.filterIsInstance(): Flow<R> = filter { it is R } as Flow<R>
 
+/**
+ * Returns a flow containing only values that are instances of the given [kClass].
+ */
+public fun <R : Any> Flow<*>.filterIsInstance(kClass: KClass<R>): Flow<R> = filter { kClass.isInstance(it) } as Flow<R>
```

Comment:
Not sure if there's some convention regarding `KClass` variable names, maybe `klass` or something?

Reply:
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/filter-is-instance.html it's `klass`

Reply:
Oh yeah, should've looked at stdlib for examples. Will fix